### ISSUE TITLE
[Beta] Applied Accessibility - Test fails to check for closing </header> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16414]
#### Challenge Name
https://beta.freecodecamp.org/en/challenges/applied-accessibility/make-screen-reader-navigation-easier-with-the-header-landmark

#### Issue Description

Test passes if you use an opening `<header>` tag and leave out an closing `</header>` tag. The test should check if the `<header>` tag is being closed.

#### Browser Information

* Browser Name, Version: Crhome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
<body>

  <header>
    <h1>Training with Camper Cat</h1>
    

  <main>
    <section id="stealth">
      <h2>Stealth &amp; Agility Training</h2>
      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
      <article><h3>No training is NP-complete without parkour</h3></article>
    </section>
    <section id="combat">
      <h2>Combat Training</h2>
      <article><h3>Dispatch multiple enemies with multithreaded tactics</h3></article>
      <article><h3>Goodbye world: 5 proven ways to knock out an opponent</h3></article>
    </section>
    <section id="weapons">
      <h2>Weapons Training</h2>
      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
    </section>
  </main>
</body>

```
#### Screenshot

![incorrect](https://user-images.githubusercontent.com/3828967/34630524-1e55de3e-f275-11e7-84ef-b4001188d027.jpg)

**Comment by anhtran1906** [comment]
@raisedadead I could take on this since I already submitted a similar PR at #16575. However, #16417 is also related to this issue and I'd like to fix these 2 issues in 1 commit. Could you advise me on how I could do this? Thank you!

**Comment by raisedadead** [comment]
Sure, will leave comments on the PR.
